.github/workflows/security-scan.yml

```diff
@@ -0,0 +1,36 @@
+name: Snyk Security Scan
+on: "push"
+#   schedule:
+#     - cron: "15 * * * *"
+# jobs:
+#   frontend-scan:
+#     name: frontend scan
+#     runs-on: ubuntu-latest
+#     steps:
+#       - uses: actions/checkout@master
+#       - run: cd frontend
+#       - name: Run Snyk to check for vulnerabilities
+#         uses: snyk/actions/node@master
+#         env:
+#           SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+#   backend-scan:
+#     name: backend scan
+#     runs-on: ubuntu-latest
+#     steps:
+#       - uses: actions/checkout@master
+#       - run: cd backend
+#       - name: Run Snyk to check for vulnerabilities
+#         uses: snyk/actions/python@master
+#         env:
+#           SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+jobs:
+  security-scan:
+    name: security scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/node@master
```

Comment:
This doesn't seem to be passing currently 😞 I wonder whether we need to give it a hint to run in a folder? Happy to explore further if you're interested